src/tests/ftest/cart/utils_test/set_other_env_vars.yaml

```diff
@@ -0,0 +1,13 @@
+defaultENV:
```

Comment:
Do we still need this yaml file?

Reply:
YAML file removed: 25412f9bdc942cdc01f7781a8dbe35bc901ebe7a